src/xcode/ENA/ENA/Source/Scenes/Checkin/EditCheckInTime/Views/GradientBackgroundView.swift

```diff
@@ -49,25 +51,43 @@ class GradientBackgroundView: UIView {
 		gradientView.translatesAutoresizingMaskIntoConstraints = false
 		addSubview(gradientView)
 		gradientView.type = type
+		
+		let starsView = UIImageView(image: UIImage(named: "stars"))
+		if displayStars {
+			starsView.translatesAutoresizingMaskIntoConstraints = false
+			gradientView.addSubview(starsView)
+		}
```

Comment:
adding the stars in this view will not allow all gradients to use them
As discussed earlier please add the stars to `GradientView` so all other can use it too.
Extend cases with fe `blueWihStars` you can also add the subview with images there.